commit 6eaea225c3e5ceaba5a2753248c770b218bdc4f4
Author: csagan5 <32685696+csagan5@users.noreply.github.com>
Date:   Wed Oct 11 23:12:18 2017 +0200

    [PATCH 35/60] Remove EV certificates
    From: Jan Engelhardt <jengelh@inai.de>
    Date: Thu, 2 Apr 2015 12:44:23 +0200
    
    The team chose to let EV certificates appear just like normal
    certificates. The web of trust is considered a failure in itself, so
    do not give users a false sense of extra security with EV certs.
    Instead, let them appear just like regular ones.

diff --git a/net/cert/ev_root_ca_metadata.cc b/net/cert/ev_root_ca_metadata.cc
index 3e89bb3..9bb411b 100644
--- a/net/cert/ev_root_ca_metadata.cc
+++ b/net/cert/ev_root_ca_metadata.cc
@@ -46,6 +46,16 @@ struct EVMetadata {
 
 // These certificates may be found in net/data/ssl/ev_roots.
 static const EVMetadata ev_root_ca_metadata[] = {
+  // need some dummy thing to make compiler happy, because
+  // arraysize() is implemented as a convoluted template rather than
+  // the traditional sizeof(x)/sizeof(*x)
+  { { { 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff,
+        0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff } },
+    {
+      "0",
+    }
+  },
+#if 0
     // AC Camerfirma S.A. Chambers of Commerce Root - 2008
     // https://www.camerfirma.com
     {
@@ -708,7 +718,9 @@ static const EVMetadata ev_root_ca_metadata[] = {
           0xd2, 0x09, 0xb7, 0x37, 0xcb, 0xe2, 0xc1, 0x8c, 0xfb, 0x2c, 0x10,
           0xc0, 0xff, 0x0b, 0xcf, 0x0d, 0x32, 0x86, 0xfc, 0x1a, 0xa2}},
         {"2.16.840.1.114404.1.1.2.4.1", ""},
-    }};
+    }
+#endif
+};
 
 #endif  // defined(USE_NSS_CERTS) || defined(OS_IOS) || defined(OS_WIN)
 
